.gitlab-ci.yml

```diff
@@ -9,6 +9,16 @@ stages:
 # Tests
 #-----------------
 
+
+check_tag:
+  stage: test
+  only:
+    - tags
+  tags:
+    - factory-plain
+  script:
+    - ./tests/check_tag.sh ${CI_COMMIT_REF_NAME}
+
```

Comment:
~~Que fait cette tache ?~~

Reply:
This is not going to work `CI_COMMIT_REF_NAME` is not the tag: it may be any branch name.

Reply:
You assume that the tag is be related to the version and to the branche name, this is asolutely not the case.

Reply:
But this test is executed only for tags:

```
  only:
    - tags
```

Reply:
Arf, my bad. did not catch that.

Reply:
No problem.
This test is also on other projects :  
* AtlasPrint https://github.com/3liz/qgis-atlasprint/blob/master/.gitlab-ci.yml#L14..L22
* Py-QGIS-Server https://github.com/3liz/py-qgis-server/blob/master/.gitlab-ci.yml#L33..L40
* ...

AFAIK, it's OK on these other projects.

Does it work for you (if you can approve the PR) ?

Reply:
Please remove that gitlab-ci job from the PR: it has nothing to do with the github release process.